framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -228,18 +228,7 @@ private boolean hasNullType(Map<TypeVariable, AnnotatedTypeMirror> inferredArgs)
         return boxedArgs;
     }
 
-    @Override
-    public void adaptMethodType(
-            AnnotatedTypeFactory typeFactory,
-            ExpressionTree invocation,
-            AnnotatedExecutableType methodType) {
-        // do nothing
-    }
-    // TODO: THIS IS A BIG VIOLATION OF Single Responsibility and SHOULD BE FIXED, IT IS SOLELY HERE
-    // TODO: AS A TEMPORARY KLUDGE BEFORE A RELEASE/SPARTA ENGAGEMENT
-    // TODO: TypeArgumentInference should only have an infer method (its sole responsibility)
-    // TODO: Subclasses should NOT be able to call adaptMethodType and getArgumentTypes (getArgumentTypes should be inlined)
```

Comment:
Alternatively, I could move the code in this method into `TypeArgInferenceUtil.treesToTypes` and rename that method to `getArgumentTypes`.  Then both  `DefaultTypeArgumentInference#inferTypeArgs` and `InferenceTypeArgumentInference#inferTypeArgs` could call that method.